src/components/CameraError/index.tsx

```diff
@@ -31,7 +32,10 @@ export default class CameraError extends Component<Props, State> {
 
   trackFallbackClick = (): void => {
     if (this.props.error.type === 'warning') {
-      this.props.trackScreen('fallback_triggered')
+      const properties: AnalyticsEventProperties = {
```

Comment:
1) Don't send an empty error object if there is not error
2) These kinds of "errors" are send as "ui_alerts: { [name]: [type] }" See Google sheets (for example `FACE_VIDEO_CAPTURE_ERROR`)

Reply:
1. I'm not sure I understand. In this branch, error cannot be empty 🤔
2. Yes, I did notice that and it's already filterable in our SDK Analytics Looker Explore, which is nice. However, at this point, the only prop I have is an `ErrorProp`, whereas `ui_alerts` expects a `UIAlerts` map. Would it be fine if I implement a function to convert a `ErrorProp` to a `UIAlerts` map? I could then get rid of this `error` property I'm adding and match the existing convention that you suggest. The other option would be to rename all the `ErrorNames` values, but I'm afraid this had non-trivial impact.